lib/main.js

```diff
@@ -60,6 +60,7 @@ export default {
     const Composer = require('./composer')
 
     global.latex = new Latex()
+    latex.opener.initializeOpeners()
```

Comment:
Can we move this to the `Latex` constructor if/when we make `initializeOpeners` synchronous? Feels like some responsibility has leaked out here — this needs to be done either in `Latex` or `OpenerRegistry`.